pandas/core/indexes/datetimes.py

```diff
@@ -213,6 +213,12 @@ class DatetimeIndex(DatelikeOps, TimelikeOps, DatetimeIndexOpsMixin,
            Attempt to infer fall dst-transition hours based on order
     name : object
         Name to be stored in the index
+    dayfirst : bool, default False
+        If True, parse dates in `data` with the day first order
```

Comment:
you can remove the see....from are instead add an entry in See Also to to_datetime

Reply:
@jreback, done